pandas/tests/frame/test_query_eval.py

```diff
@@ -495,7 +495,7 @@ def test_query_syntax_error(self):
             df.query("i - +", engine=engine, parser=parser)
 
     def test_query_scope(self):
-        from pandas.core.computation.ops import UndefinedVariableError
```

Comment:
These imports might be able to be moved to the top (in a follow up PR would be okay)